cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,96 @@
+    cy.get('button[type="submit"]').click();
+
+    cy.contains('Invalid login credentials. Please try again').should('be.visible');
+  });
+
+  it('should login successfully with the correct credentials', () => {
+    cy.get('#email').type('cypress-test@noroff.no');
+    cy.get('#password').type('czv4euj*ncv6NUG@aqy');
+    cy.get('button[type="submit"]').click();
+
+    cy.get('#profileName').should('contain', 'cypress test');
+    cy.get('#profileRole').should('contain', 'Applicant');
+    cy.get('#profileEmail').should('contain', 'cypress-test@noroff.no');
+  });
+
+  it('should logout successfully', () => {
+    cy.get('#email').type('cypress-test@noroff.no');
+    cy.get('#password').type('czv4euj*ncv6NUG@aqy');
+    cy.get('button[type="submit"]').click();
+
+    cy.get('#profileName').should('contain', 'cypress test');
+    cy.get('#profileRole').should('contain', 'Applicant');
+    cy.get('#profileEmail').should('contain', 'cypress-test@noroff.no');
+
+    cy.contains('Log out').click();
+
+    cy.contains('Welcome to Noroff Job Agency').should('be.visible');
+
+    cy.checkLocalStorage('email', null);
+    cy.checkLocalStorage('id', null);
+    cy.checkLocalStorage('role', null);
+    cy.checkLocalStorage('token', null);
+
+    cy.checkSessionStorage('email', null);
+    cy.checkSessionStorage('id', null);
+    cy.checkSessionStorage('role', null);
+    cy.checkSessionStorage('token', null);
+  });
+
+  it('should save credentials to local storage when remember is checked', () => {
+    cy.get('#email').type('cypress-test@noroff.no');
+    cy.get('#password').type('czv4euj*ncv6NUG@aqy');
+    cy.get('input[type="checkbox"]').check();
+    cy.get('button[type="submit"]').click();
+
+    cy.get('#profileName').should('contain', 'cypress test');
+    cy.get('#profileRole').should('contain', 'Applicant');
+    cy.get('#profileEmail').should('contain', 'cypress-test@noroff.no');
+
+    cy.checkLocalStorage('email', 'cypress-test@noroff.no');
+    cy.checkLocalStorage('id', '649e0993-ce87-474e-8714-9e44d2de8e40');
+    cy.checkLocalStorage('role', 'Applicant');
+    cy.checkLocalStorage('token', 'NOT_NULL');
+
+    cy.checkSessionStorage('email', null);
+    cy.checkSessionStorage('id', null);
+    cy.checkSessionStorage('role', null);
+    cy.checkSessionStorage('token', null);
+  });
+
+  it('should save credentials to session storage when remember is not checked', () => {
+    cy.get('#email').type('cypress-test@noroff.no');
+    cy.get('#password').type('czv4euj*ncv6NUG@aqy');
+    cy.get('input[type="checkbox"]').uncheck();
+    cy.get('button[type="submit"]').click();
+
+    cy.get('#skillsHeader').should('contain', 'Skills');
+
+    // profile view is currently broken when using session storage, e.g "don't remember me"
+
+    // cy.get('#profileName').should('contain', 'cypress test');
+    // cy.get('#profileRole').should('contain', 'Applicant');
+    // cy.get('#profileEmail').should('contain', 'cypress-test@noroff.no');
```

Comment:
I can understand keeping this commented out code, since it might be fixed and can be re-activated without having to type it out again. BUT, for me it looks out of place, and considering how many working on the repo / branch, names and such might change.
I'm not forcing a change request on it, but i think it's enough to keep the comment on line 80 only.

Reply:
fair point, i've remove the code and updated the comment